code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -799,6 +799,16 @@
 	if(prob(5))
 		M.emote(pick("twitch_v", "grunt"))
 
+	if((M.bodytemperature < 151) && M.chem_effects[CE_CRYO]) //red nightshade in extracool cryogenic conditions will restore bonebreaks, at the cost of blood depletion
+		var/mob/living/carbon/human/H = M
+		H.vessel.remove_reagent(/singleton/reagent/blood, 15)
+		for(var/obj/item/organ/external/E in H.organs)
+			if(E.status & ORGAN_BROKEN)
+				if(prob(10))
+					H.vessel.remove_reagent(/singleton/reagent/blood, 30)
+					E.status &= ~ORGAN_BROKEN
+					M.visible_message("<b>[M]</b>'s spasms!", SPAN_DANGER("You feel a stabbing pain!"))
```

Comment:
This seems a bit out of nowhere, why is this being added?

Reply:
Healing bonebreaks without surgery is pretty strong, so I tied it to a drug with high consequence use. In an absolute emergency you can use nightshade in cryogenic conditions, it will lead to extended patient care being required until the patient calms down as well as being able to justify it's use should any command ask why you used an illegal drug. 

Why Nightshade though? It has strength enhancing qualities, probably affects the muscoskeletal system to some degree by increasing muscle/bone density/blah. In a controlled setting in cryo, the good spin is you can harness that and use it to repair bone breaks.

Why bone healing at all? The surgeonless rounds where someone has a head, chest or groin fracture. These can't be splinted and otherwise condemn someone to a round in a wheelchair. Most rounds this is fine, but if there's some Code Red situation going on and all the surgeons are in residential/dead, Red Nightshade cryotherapy as a desperate means to get people capable of walking unsupported again would be high risk-high reward.

Reply:
Ehhhhhhhhhhhhhhhhhhhhhhhhhh. I don'tknow. I need to think about it. Implement the other changes then I can probably testmerge this.

Reply:
Just a note, physicians can fix broken bones via surgery per surgery chart, so even if there's no surgeon on shift, as long as there's a physician this is not an issue